python/test/unit/language/test_reproducer.py

```diff
@@ -30,7 +34,12 @@ def test_reproducer():
             foundPipeline = line
     if 0 == len(foundPipeline):
         raise Exception("Failed to find pipeline info in reproducer file.")
-    if "convert-triton-gpu-to-llvm" not in foundPipeline:
+    ttgir_to_llvm_pass = "convert-triton-"
+    if is_hip():
+        ttgir_to_llvm_pass += "amd"
+    ttgir_to_llvm_pass += "gpu-to-llvm"
```

Comment:
let's just check something like `convert-triton-{{.*}}gpu-to-llvm` it doesn't look great to have to do something specific to AMD in this test

Reply:
Thanks. Fixed.